source/operations/checklists/hardware.rst

```diff
@@ -202,37 +202,54 @@ Storage
 
 .. cond:: not k8s
 
-   MinIO recommends selecting the type of drive based on your performance objectives.
-   The following table highlights the general use case for each drive type based on cost and performance:
+   MinIO recommends using flash-based storage (NVMe or SSD) for all workload types and scales.
+   Workloads that require high performance should prefer NVMe over SSD.
 
-   .. list-table::
-      :header-rows: 1
-      :widths: auto
-      :width: 100%
+   MinIO deployments using HDD-based storage are best suited as cold-tier targets for :ref:`Object Transition ("Tiering") <minio-lifecycle-management-tiering>` of aged data.
+   HDD storage typically does not provide the necessary performance to meet the expectations of modern workloads, and any cost efficiencies at scale are offset by the performance constraints of the medium. 
 
-      * - Type
-        - Cost
-        - Performance
-        - Tier
+   Format Drives as XFS
+   ++++++++++++++++++++
 
-      * - NVMe
-        - High
-        - High
-        - Hot
+   Format drives as XFS and present them to MinIO as a :abbr:`JBOD (Just a Bunch of Disks)` array with no RAID or other pooling configurations.
 
-      * - SSD
-        - Balanced
-        - Balanced
-        - Hot/Warm
+   MinIO **strongly recommends** disabling retry-on-error (``EIO``, ``ENOSPC``, and ``default``) behavior in XFS.
+   The default settings typically allow infinite retry-on-error and can produce unnecessary performance loss.
 
-      * - HDD
-        - Low
-        - Low
-        - Cold/Archival
+   The following script iterates through all drives at the specified mount path and sets recommended values for ``max_retries`` and ``retry_timeout_seconds``. 
```

Comment:
@eco-minio @harshavardhana 

Tested this on RHEL9 via cronjob and it worked through multiple restarts.

We can `cat /var/log/cron` to see command outputs, and that should include all drives touched by the script. Might help in debugging.

It does require all drives to be mounted via fstab though.